### ISSUE TITLE
Grid advanced search button doesn't work on localhost

### ISSUE THREAD

**Issue by avber** [vitmalina/w2ui#1419]
IE 11

http://localhost:8080/demos/#!grid/grid-17
Advanced search dialog is shown only after the 1st click on the button. If you click again, the dialog is hidden. If you click the button once again, nothing happens.

Multiple clicks work correctly on the website.
http://w2ui.com/web/demos/#!grid/grid-17

**Comment by mpf82** [comment]
Possible duplicate of #1409

**Comment by avber** [comment]
>I guess you're using the latest sources from github

right

There is another issue with the button's overlay. Suppose the search button is located at the right side of a grid's toolbar and no buttons come after it. When the overlay appears, document body scrollbars are shown briefly. Overlay size and position should be calculated before overlay is shown.

**Comment by mpf82** [comment]
Seems to be fixed, can no longer be reproduced in [fiddle](http://jsfiddle.net/ya9eerb5/9/)

Please close, unless you can still reproduce the issue.
